packages/docs/content/migration-guides/standardize-theme-colors.mdx

```diff
@@ -407,15 +410,22 @@ Coolblue was unused in the design system and is removed.
 
 ### Alert 
 
-The alert component has been updated to use `info` colors for the left border color and background color.
+The alert component has been updated to use `info` colors for the left border color and background color. This change does not result in visual changes.
 
 ### Calendar picker button
 
 Previously, `gray-cool-light` was mapped to the same system color token for all themes.
 
 When removing `gray-cool-light`, we remapped the background color for the hover state of [calendar picker](/components/date-field/date-picker/) buttons to `primary-lightest`. On all but the core theme, this results in a slight color shift for the background color of those day-picker buttons when hovered.
 
+<ThemeContent onlyThemes={['core']}>
 
+<ColorMigrationTable
+  colorRows={[
+    [{ name: 'ocean-50', value: '#e6f1f8' }, { name: 'sky-50', value: '#e6f9fd' }],
```

Comment:
@pwolfert - this section for the calendar picker button only shows both color names and seems kinda weird. Maybe there is a setting to change or something to only show one instance of the color name?

Reply:
We could instead show the theme variable name now that we're at peace with much longer names in here

Reply:
Also this section added isn't accurate. Core didn't change. It stayed `ocean-50`

Reply:
This is what it would look like

<img width="878" alt="Screenshot 2023-03-03 at 3 15 21 PM" src="https://user-images.githubusercontent.com/7595652/222852936-38181887-9832-43e8-89cd-9587fc67f69f.png">

Reply:
Nice, I'm OK with the longer name if you are :)

Reply:
I've been thinking about showing the change we made to the drawer footer background color (which is the same), and I think it would make more sense to compare theme-level variables and show `--color-primary-lightest` on the right instead of `--day-picker-button__background-color--hover`. It would be more like comparing apples to apples.